api.swagger.yaml

```diff
@@ -1845,6 +1845,36 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/pdf-export/{node_id}':
+    get:
+      description: Get a node in HTML for PDF export.
+      tags:
+        - pdf-export
+      parameters:
+        - name: node_id
```

Comment:
I am being silly, we don't use _uuid - I was just confused because node_id looks like it implies the drupal nid, but elsewhere we've used the content type. This is not content type specific but I don't think "node" is the ideal term to use either. How about `content_id`?

Reply:
Change applied.